ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/http/LogErrorHandler.java

```diff
@@ -26,7 +26,7 @@
   public void handle(final RoutingContext failureContext) {
 
     if (failureContext.failed()) {
-      LOG.error(
+      LOG.debug(
```

Comment:
Think we can just get rid of the LogErrorHandler all together. As the JsonRpcErrorHandler will log the same details and is also registered as a failureHandler